pulp_container/app/access_policy.py

```diff
@@ -43,3 +44,45 @@ def has_namespace_obj_perms(self, request, view, action, permission):
         """
         namespace = view.get_object().namespace
         return request.user.has_perm(permission, namespace)
+
+
+class ContainerContentAccessPolicyFromDB(AccessPolicyFromDB):
+    """
+    Access policy for Tags/Manifests/Blobs viewsets.
+    """
+
+    def has_repo_query_params(self, request, view, mirror_repo, push_repo):
```

Comment:
I'm in doubt, that this function can be used in the access policies statements. Not sure whether that's a reason to move it elsewhere, though.